commons/passive/dreem/dreem_sleep_endpoints.avsc

```diff
@@ -17,53 +17,121 @@
+    },
+    {
+      "name": "N3Endpoint",
+      "type": "float",
+      "doc": "N3 stands for NREM 3 duration. Unit is min. It is minutes of N3 sleep. Value associated with the endpoint. Format: XX.XX"
     },
     {
-      "name": "qiIndex",
+      "name": "REMEndpoint",
+      "type": "float",
+      "doc": "REM stands for Total Rapid Eye Movement Sleep duration. Unit is min. It is minutes of REM sleep. Value associated with the endpoint. Format: XX.XX"
+    },
+    {
+      "name": "p_N1Endpoint",
+      "type": "float",
+      "doc": "p_N1 stands for percentage Stage N1 Sleep. Unit is %. It is percentage of TST spent in N1 sleep. Value associated with the endpoint. Format: XX.XX"
+    },
+	{
+      "name": "p_N2Endpoint",
+      "type": "float",
+      "doc": "p_N2 stands for percentage Stage N2 Sleep. Unit is %. It is percentage of TST spent in N2 sleep. Value associated with the endpoint. Format: XX.XX"
+    },
+    {
+      "name": "p_N3Endpoint",
+      "type": "float",
+      "doc": "p_N3 stands for percentage Stage N3 Sleep. Unit is %. It is percentage of TST spent in N3 sleep. Value associated with the endpoint. Format: XX.XX"
+    },	
+    {
+      "name": "p_REMEndpoint",
+      "type": "float",
+      "doc": "p_REM stands for Percentage Stage REM Sleep. Unit is %. It is percentage of TST spent in REM sleep. Value associated with the endpoint. Format: XX.XX"
+    },
+    {
+      "name": "p_NREMEndpoint",
+      "type": "float",
+      "doc": "p_NREM stands for Total Sleep NREM Sleep/ Total Sleep Time. Unit is %. It is percentage of TST spent in NREM sleep (N1 + N2 + N3). Value associated with the endpoint. Format: XX.XX"
+    },
+    {
+      "name": "RREndpoint",
+      "type": "float",
+      "doc": "RR stands for Mean Respiratory rate. Unit is BPM. Is is average number of breaths per minute through the recording. Value associated with the endpoint. Format: XX.XX"
+    },
+    {
+      "name": "RRN1Endpoint",
+      "type": "float",
+      "doc": "RRN1 stands for respiratory rate during N1. Unit is BPM. It is average number of breaths per minute in N1 sleep. Value associated with the endpoint. Format: XX.XX"
+    },
+    {
+      "name": "RRN2Endpoint",
+      "type": "float",
+      "doc": "RRN2 stands for respiratory rate during N2. Unit is BPM. It is average number of breaths per minute in N2 sleep. Value associated with the endpoint. Format: XX.XX"
+    },
+    {
+      "name": "RRN3Endpoint",
+      "type": "float",
+      "doc": "RRN3 stands for respiratory rate during N3. Unit is BPM. It is average number of breaths per minute in N3 sleep. Value associated with the endpoint. Format: XX.XX"
+    },
+    {
+      "name": "QUALEndpoint",
+      "type": "float",
+      "doc": "QUAL stands for recording quality. Unit is %. It is proportion of recording that is unequivocally scorable by a sleep expert. Value associated with the endpoint. Format: XX.XX"
+    },
+    {
+      "name": "confidenceIndex",
```

Comment:
If I understand correctly, a separate confidence index is needed for each endpoint. I would change the endpoint types to a new file with contents
```json
{
    "namespace": "org.radarcns.passive.dreem"
    "name": "DreemMetric",
    "type": "record",
    "doc": "Single computed Dreem endpoint Metric."
    "fields": [
        {"name": "value", "type": "float", "doc": "Value of the given metric"},
        {"name": "confidenceIndex": "doc": "Confidence index etc..."}
     ]
}
```